lib/active_merchant/billing/gateways/paymentez.rb

```diff
@@ -34,7 +34,8 @@ class PaymentezGateway < Gateway #:nodoc:
         28 => :card_declined
       }.freeze
 
-      SUCCESS_STATUS = ['success', 'pending', 1, 0]
+      SUCCESS_STATUS = ['APPROVED', 'PENDING', 1, 0]
+      FAILURE_STATUS = ['failure', 'Internal server error']
```

Comment:
I did some tests but I couldn't get: 'Internal server error'. How can we obtain this result?

Reply:
The specific unit test ´test_paymentez_crashes_fail´ gives that result.